paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/TableNameConverter.java

```diff
@@ -58,4 +71,18 @@ public String convert(Identifier originIdentifier) {
                                 + originIdentifier.getObjectName();
         return convert(rawName);
     }
+
+    private Map<String, String> lowerMapKey(Map<String, String> map) {
+        int size = map == null ? 0 : map.size();
```

Comment:
if map == null, then return "Collections.emptyMap()".
I think this is better.

Reply:
I'd like insist. :-)